app/plugin_api.go

```diff
@@ -70,6 +71,19 @@ func (api *PluginAPI) UnregisterCommand(teamId, trigger string) error {
 	return nil
 }
 
+func (api *PluginAPI) ExecuteSlashCommand(commandArgs *model.CommandArgs) (*model.CommandResponse, error) {
+	user, err := api.app.GetUser(commandArgs.UserId)
+	if err != nil {
+		return nil, err
+	}
+	commandArgs.T = utils.GetUserTranslations(user.Locale)
```

Comment:
Should the server take care of setting 'commandArgs.SiteURL`?

Reply:
Ya. I think so. Will fix.